src/sources/http.rs

```diff
@@ -199,7 +199,7 @@ fn body_to_lines(buf: Bytes) -> impl Iterator<Item = Result<Bytes, ErrorMessage>
     })
 }
 
-fn decode_body(body: Bytes, enc: Encoding) -> Result<Vec<Event>, ErrorMessage> {
+pub fn decode_body(body: Bytes, enc: Encoding) -> Result<Vec<Event>, ErrorMessage> {
```

Comment:
Could we pull this out into `src/sources/util/http.rs` since we are sharing it now? That should let you drop the feature dependency in Cargo.toml too.

Reply:
I relocated the code and adjusted Cargo.toml accordingly.